source/nodejs/adaptivecards-site/themes/adaptivecards/scripts/helper-properties-details.js

```diff
@@ -20,7 +20,12 @@ hexo.extend.helper.register('properties_details', function (locals, properties,
 		// Get the markdown for the property and turn it into HTML
 		html += '<div class="w3-container">';
 
-		html += marked(typedschema.markdown.createPropertyDetails(property, 3, null, false, true, elementVersion), { headerPrefix: "dedupe-header" });
+		// mark header with class name like ac-schema-version-1.5
+		html +=  `<div class="ac-schema-version-${elementVersion?.replace(/\./, '-')}" style="display: flex;">`
```

Comment:
nit: why we don't refactor this as we do in createPropertyDetailsHeader as etc?

Reply:
The problem is `marked` function, `div` is converted to `%3Cdiv%3E`, so it's not a `<div>` anymore.
markdown has special handle for `<span>` but it has side effect that it impacted layout. It's only be expected to use in table of markdown.